package.json

```diff
@@ -79,7 +79,7 @@
     "build:integration:ssr": "yarn build:integration && nx server integration --configuration production",
     "// - TESTING": "Test all modules #requires yarn build:packages",
     "test:integration": "nx run integration:test --colors --run-in-band",
-    "test:types": "dtslint packages/store/types --localTs node_modules/typescript/lib",
+    "test:types": "cd packages/store/types && yarn test:types",
```

Comment:
Something to do with this change seems to be failing the tests.
I see that there is a `package.json` file with a `yarn.lock`but no install is run from ci for this folder.
- Potentially we could add a `workspaces` property in the main `package.json` that includes this folder to make that happen, but I'm not sure how `yarn` does it, compared to modern `npm`.
- Or we could modify the `.github/actions/setup/action.yml` file to also run the install from that folder.
- Or we could add a `yarn install... ` command into the types folder's `package.json` to do the install just in time. Just like how we do for the integration apps.